docs/reference-guide/metadata.md

```diff
@@ -0,0 +1,88 @@
+---
+id: metadata
+title: Metadata
+sidebar_label: Metadata
+---
+
+Bolt protocol specifies additional data that can be sent along with the
+requested results. Such data is called metadata and can be divided into two
+groups:
+ - Query Statistics,
+ - Notifications.
+
+Both of these metadata can be accessed through `summary` map that is being sent
+along with the results of the query. Query statistics will be under `stats` key,
+and notifications under `notifications` key.
+
+## Query Statistics
+
+Query statistics will be sent whenever a user performs a query that will effect
+data in any way. In other words we will track the quantity of these changes
+throughout the query execution and report it back to the user.
+
+The structure of statistics is a map of string keys and integer values. Data
+that is being tracked:
+
+ - `nodes-created`
+ - `nodes-deleted`
+ - `relationships-created`
+ - `relationships-deleted`
+ - `labels-added`
+ - `labels-removed`
+ - `properties-set`
+
+This data will refer only to the changes done by query, not triggers changes will
+not affect these values.
+
+If the query changes and deletes nodes like this `CREATE (n) DELETE n;` and
+leaves memgraph in the same state as before. The results will be 1 for both
+`nodes-created` and `nodes-deleted` but the triggers will not be triggered since
+there were no changes across the transaction.
```

Comment:
What about this?

```suggestion
:::caution Differences compared to triggers

It is possible that after executing a query some of these counters are not
zero, however the regarding triggers are not triggered. The reason for that is
triggers are only  triggered when there is a difference between the starting
and ending state, while the counters are also counting the not permanent
changes.

For example if the query changes and deletes nodes like
`CREATE (n) DELETE n;`, then it leaves Memgraph in the same state as before.
The value will be 1 for both `nodes-created` and `nodes-deleted`, but the
triggers will not be triggered since there were no changes across the
transaction.

:::
```

I think it gives a bit more general picture about how the counters and triggers differ while keeping the example for providing something concrete

Edit: I just saw the note from @antonio2368 and it makes this a bit outdated, but as I already wrote this, I think it is better to publish it.